site/en/docs/devtools/console/log/index.md

```diff
@@ -188,8 +160,6 @@ message.
 
     {% Img src="image/admin/t1U5iAP8oKCS0nlwaOZy.png", alt="Disabling Error-level messages in the Console.", width="800", height="518" %}
```

Comment:
Update "Filter by log level" session to include link to the severity like what you did in console reference.